app/containers/container_dropNewPin.js

```diff
@@ -1,7 +1,46 @@
 //this should add ownprops from parent map
-import dropNewPin from '../actions/action_pins.js';
+import React, { Component } from 'react-native';
+import getLocationToSave from '../actions/action_pins.js';
+import Button from 'react-native-button';
 
+// import DropNewPinButton from '../components/DropNewPinButton.js';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+class DropNewPinButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state= {
+     long: null,
+     lat: null,
+     title: ''
+    };
+  }
+
+  handleClick() {
+   this.props.actions.getLocationToSave();
+  }
+  render() {
+    return (
+      <Button
+        style={{borderWidth: 1, borderColor: 'blue'}}
+        onPress={this.handleClick.bind(this)}>
+        Press Me!
+      </Button>
```

Comment:
This button looks like it is working as intended, and is altering the state to add new pin data